dev/2018/spring/knights-to-kings/knights-to-kings.html

```diff
@@ -0,0 +1,196 @@
+<article class="story story-default">
+  <div class="container">
+    <div class="row">
+        <img src="[image filename='Knights-to-Kings-UCF-Football']" alt="From Knights to Kings" title="From Knights to Kings" class="img-responsive center-block story-header">
```

Comment:
Since we're not using a video here in the header like we were planning on originally, we could improve the SEO of the h1 by wrapping the image inside of it:

`<h1><img src="[image filename='Knights-to-Kings-UCF-Football']" alt="From Knights to Kings: UCF Football players reflect on the 2017 season" title="From Knights to Kings" class="img-responsive center-block story-header"></h1>`

The image's alt text will serve as the readable h1 content.